smirnoff99Frosst/smirnoff99Frosst.ffxml

```diff
@@ -276,24 +276,24 @@
-    <Proper smirks="[#8X1:1]~[#16X4,#16X3+0:2]-[#7X2:3]~[#6X3:4]" id="t148" idivf1="1" k1="0.071" periodicity1="6" phase1="0.0" k2="0.697" phase2="0.0" idivf2="1" periodicity2="5" phase3="180.0" idivf3="1" k3="0.208" periodicity3="4" periodicity4="2" k4="3.931" phase4="180.0" idivf4="1" periodicity5="3" idivf5="1" k5="1.940" phase5="180.0" idivf6="1" k6="2.448" periodicity6="1" phase6="0.0"/>
+    <Proper smirks="[#6X4:1]-[#16X4,#16X3+0:2]-[#7X2:3]~[#6X3:4]" id="t147" idivf1="1" k1="0.040" periodicity1="6" phase1="0.0" k2="0.407" idivf2="1" periodicity2="5" phase2="0.0" phase3="0.0" k3="0.013" idivf3="1" periodicity3="4" idivf4="1" phase4="0.0" periodicity4="3" k4="0.018" periodicity5="2" idivf5="1" phase5="180.0" k5="1.329" phase6="0.0" idivf6="1" periodicity6="1" k6="0.927"/>
+    <Proper smirks="[#8X1:1]~[#16X4,#16X3+0:2]-[#7X2:3]~[#6X3:4]" id="t148" idivf1="1" k1="0.071" periodicity1="6" phase1="0.0" k2="0.697" idivf2="1" periodicity2="5" phase2="0.0" phase3="180.0" k3="0.208" idivf3="1" periodicity3="4" idivf4="1" phase4="180.0" periodicity4="2" k4="3.931" periodicity5="3" idivf5="1" phase5="180.0" k5="1.940" phase6="0.0" idivf6="1" periodicity6="1" k6="2.448"/>
     <Proper smirks="[*:1]~[#16X4,#16X3+0:2]-[#8X2:3]-[*:4]" id="t149" idivf1="1" k1="0.000" periodicity1="1" phase1="0.0"/>
-    <Proper smirks="[*:1]-[#16X2,#16X3+1:2]-[#16X2,#16X3+1:3]-[*:4]" id="t150" idivf1="1" k1="3.500" periodicity1="2" phase1="0.0" k2="0.600" phase2="0.0" idivf2="1" periodicity2="3"/>
+    <Proper smirks="[*:1]-[#16X2,#16X3+1:2]-[#16X2,#16X3+1:3]-[*:4]" id="t150" idivf1="1" k1="3.500" periodicity1="2" phase1="0.0" k2="0.600" idivf2="1" periodicity2="3" phase2="0.0"/>
     <Proper smirks="[*:1]-[#8X2:2]-[#15:3]~[*:4]" id="t151" idivf1="1" k1="0.750" periodicity1="3" phase1="0.0"/>
-    <Proper smirks="[#8X2:1]-[#15:2]-[#8X2:3]-[#6X4:4]" id="t152" idivf1="1" k1="0.250" periodicity1="3" phase1="0.0" k2="1.200" phase2="0.0" idivf2="1" periodicity2="2"/>
+    <Proper smirks="[#8X2:1]-[#15:2]-[#8X2:3]-[#6X4:4]" id="t152" idivf1="1" k1="0.250" periodicity1="3" phase1="0.0" k2="1.200" idivf2="1" periodicity2="2" phase2="0.0"/>
     <Proper smirks="[*:1]~[#7X3:2]-[#15:3]~[*:4]" id="t153" idivf1="1" k1="2.5" periodicity1="2" phase1="180.000"/>
     <Proper smirks="[*:1]~[#7X4:2]-[#15:3]~[*:4]" id="t154" idivf1="1" k1="0.1" periodicity1="3" phase1="0.000"/>
-    <Proper smirks="[*:1]-[#7:2]-[#15:3]=[*:4]" id="t155" idivf1="1" k1="3.000" periodicity1="2" phase1="180.0" k2="2.300" phase2="0.0" idivf2="1" periodicity2="3"/>
+    <Proper smirks="[*:1]-[#7:2]-[#15:3]=[*:4]" id="t155" idivf1="1" k1="3.000" periodicity1="2" phase1="180.0" k2="2.300" idivf2="1" periodicity2="3" phase2="0.0"/>
```

Comment:
@davidlmobley 
Did something change in the openforcefield code (or the conversion script) that caused all these torsions to be printed differently? I didn't look at every single line, but they appear to just be showing the parameters in a different order so I'm fine to merge as is.